From 41b4d8ecc6e2604dd949788800a8f6f930e8ab46 Mon Sep 17 00:00:00 2001
From: "Yue, VincentX" <vincentx.yue@intel.com>
Date: Wed, 7 Mar 2018 17:59:46 +0800
Subject: [PATCH] Fix home touch buttons are not available on the navigationBar

The abnormal activity layout use a extra theme: @android:style/Theme.DeviceDefault.Settings
to set the navigationBar to white, but if the size is 1920x1080,
density is 160 and sdkversion is lower,the display will not 16:9.
It leades to the activity layout content don't covered the systemUI
navigationBar. So, we see a black navigationBar.

Due to google has fixed at the higher sdk version,
so change it to current SDK version 27.

Change-Id: I245cebf64e24d47d9aa6289ed80adfe409081fd7
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-58550
Signed-off-by: Yue, VincentX <vincentx.yue@intel.com>
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 23bddf6..295544e 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -20,7 +20,7 @@
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.android.launcher3">
-    <uses-sdk android:targetSdkVersion="23" android:minSdkVersion="21"/>
+    <uses-sdk android:targetSdkVersion="27" android:minSdkVersion="21"/>
     <!--
     Manifest entries specific to Launcher3. This is merged with AndroidManifest-common.xml.
     Refer comments around specific entries on how to extend individual components.
-- 
1.9.1

